test/e2e/app-dir/router-autoscroll/router-autoscroll.test.ts

```diff
@@ -153,7 +168,7 @@ describe('router autoscrolling on navigation', () => {
           return (
             content +
             `
-      \\\\ Add this meaningless comment to force refresh
+      // Add this meaningless comment to force refresh
```

Comment:
This wasn't a comment and just triggered a build error polluting test assertions i.e. fixes
```
Error:   × Expected unicode escape
    ╭─[/Users/sebbie/repos/next.js/test/e2e/app-dir/router-autoscroll/app/[layoutPaddingWidth]/[layoutPaddingHeight]/[pageWidth]/[pageHeight]/[param]/page.tsx:14:1]
 11 │   )
 12 │ }
 13 │ 
 14 │ \\\\ Add this meaningless comment to force refresh
    · ▲
    ╰────
```